### DISCUSSION TITLE
[省流]增加绳网等级的方法

### DISCUSSION THREAD

**Discussion by HeroBrine-c** [share121/inter-knot#1668] [其他]
### 不永久的方法

**1.F12打开开发工具**
![601df869ea7b845d8888b901b6e605a](https://github.com/user-attachments/assets/d601846d-babe-47d2-8c69-04a46d8118b1)
当然也可以鼠标右键打开检查

**2.找到大圆圈处，点开小圆圈处的三个点(如图)**
![6399e2af86ccff30ac5a85064bfb01a](https://github.com/user-attachments/assets/f7959e44-1273-441b-97a3-8c3f93decef8)

**3.找到这一行，更改圆圈处的数字就行了**
![59a6d7854fa08ec9510cc08c92143a7](https://github.com/user-attachments/assets/09c0cb40-5085-43e6-9528-3de528f51b11)

_注：刷新后会还原回原有等级_

### 以下是永久的方法

**1.创建新仓库**
**2.刷新绳网就行了**

建得越多等级越高[doge]

**Comment by VacuolePaoo** [comment]
好好好你这么玩是吧

**Comment by Yersmagit** [comment]
主打一个我自己看着舒服

**Comment by devil233-ui** [comment]
难道你是甘蔗？
